arquillian-chameleon-container-model/src/main/resources/chameleon/default/containers.yaml

```diff
@@ -147,7 +147,7 @@
   defaultProtocol: Servlet 3.0
   exclude: *EAP_EXCLUDE
 - name: JBoss AS
-  versionExpression: 7.*
+  versionExpression: 7\.[0-1]\.[0-2]\.(.*)$
```

Comment:
This will allow the user to specify `7.1.2` which is not in central 
http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.jboss.as%22%20AND%20a%3A%22jboss-as-dist%22